pkg/node/ingress_endpoint.go

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package node
+
+const (
+	TemplateIngressEndpointID = uint64(0xfffe)
+)
+
+// GetIngressEndpointID returns the ID of the ingress endpoint for this node.
+func GetIngressEndpointID() uint64 {
```

Comment:
Are these really needed? Can't you just take `LocalNodeStore` in where it's needed and access it through that? We are trying to get rid of these global getters and setters so it's not great that new ones are added.

Reply:
Good question, not really! I had used the pattern of host EndpointID in the node package, but only now noticed that it is needed for feeding the host endpoint ID to the bpf datapath; there is no need for that for ingress endpoint ID.